libs/ui-lib/lib/common/components/ui/WizardNavItem.tsx

```diff
@@ -14,7 +14,7 @@ const getNavItemContent = (
     return (
       <>
         {content}
-        <Icon size="sm" status="danger" className="wizard-nav-item-warning-icon">
+        <Icon status="danger" className="wizard-nav-item-warning-icon">
           <ExclamationCircleIcon />
```

Comment:
Wouldn't the color also be necessary for the nav item icons?

Reply:
It seems that `<Icon status="..."` does the same thing as `<Icon><ConcreteIcon color={...}`, so it's not really needed.

Reply:
The question is, of course, do we need to specify both in the other case?

Reply:
@jgyselov @celdrake I took a look to uhc-portal and they use this format:
 const iconProps = { className: 'clusterstate', size: 'md', };
And then:
<Icon {...iconProps}> <ExclamationCircleIcon color={dangerColor.value} /> </Icon>

Maybe is better to implement in the same way in all places that we are using icons. WDYT?

Reply:
- In places where we set a particular icon as a prop of another component we need to set the color because there is no Icon component wrapping the icon itself.
- We could extract all of this into our own component that would wrap this part  and we would just provide props based on the use case:

```
const iconProps = { className: 'clusterstate', size: 'md', };
<Icon {...iconProps}> <ExclamationCircleIcon color={dangerColor.value} /> </Icon>
```